draft-ietf-oauth-transaction-tokens.md

```diff
@@ -433,6 +437,19 @@ grant_type=urn%3Aietf%3Aparams%3Aoauth%3Agrant-type%3Atoken-exchange
 ~~~
 {: #figtxtokenrequest title="Example: Txn-Token Request"}
 
+### Subject Token Types
+The `subject_token_type` parameter in the Txn-Token Request MAY have one of the following values:
+
+rfc9068
+: The `subject_token` is a JWT format OAuth 2.0 Access Token as defined in {{RFC9068}}
+
+id-token
```

Comment:
RFC 8693 already defines a `subject_token_type` value for this which we should use. That said, `id_tokens` aren't usually authorization tokens and so I worry about explicitly calling it out here. The `id_token` can most definitely define the subject but then there is likely a need to pass in a different token containing the external authorization state. I'd rather we focus on AS generated access tokens and self-signed access tokens (recommended as RFC 9068).